File: Practice.java
public class Practice {
    public static void main(String[] args) {
        // System.out.println("Hello, World!");
        int score = 100;
        if (score >= 50) {
            System.out.println("Pass!");
        } else {
            System.out.println("Fail");
        }
        System.out.println(add(2, 4));
    }
    
    public static int add(int a, int b) {
        // System.out.println(a + b);
        return a + b;
    }
}

File: practice.c
#include <stdio.h>

int add(int a, int b) {
    // printf("Hello, World\n");
    // return 0;
    // int score = 60;
    // if (score >= 50) {
    //     printf("Pass\n");
    // }else {
    //     printf("Fail\n");
    // }
    int sum = a + b;

    // printf("%d\n", sum);
    return a + b;
}

int main() {
    printf("%d\n", add(3, 4));
}